lib/kontena/templates/haproxy/http_in.text.erb

```diff
@@ -0,0 +1,41 @@
+listen http-in
+    bind *:80
+    http-request replace-header Host (.*):.* \1
+    <% if ENV['SSL_CERTS'] %>
+    bind *:443 ssl crt /etc/haproxy/certs/ no-sslv3
```

Comment:
Better to use the instance-local `ENV['SSL_CERTS']` for this, rather than going via the etcd `.../certs/bundle` key like the current confd template does.